.github/workflows/ci.yml

```diff
@@ -51,12 +51,12 @@ jobs:
 
       - name: Install dependencies
         id: install-dependencies
-        run: poetry install
+        run: poetry install --extras=all
 
       - name: Execute Python tests
         id: execute-tests
         run: |
-          poetry run python -m pytest --junit-xml=junit.xml --cov-report term --cov-report xml:coverage.xml --cov=sekoia_automation tests
```

Comment:
Why did you remove the directory `tests` in the command line?

Reply:
I removed it because of incorrect configuration between local execution and gitlab ci.
For some reason it does not resolve paths with `tests` on local env